### PR TITLE
Ignore existing warnings; fail on new

### PR DESCRIPTION
**Pull request by bf4**
Follows #1201 
- [ ] Reviewers decide if they want to keep the warnings or fix them :)

config/brakeman.yml generated via:
`brakeman -z -w2 -q -A --routes --message-limit 200 --table-width 200 \
  --github-repo rubygems/rubygems.org -4 -i config/brakeman.ignore -d -p .  --summary \
  --skip-files config/database.yml --safe-methods banana \
  --url-safe-methods banana_url --compare reports/brakeman.json -o \
  /dev/stdout -o reports/brakeman.json -o reports/brakeman.html -C &> \
  config/brakeman.yml`

config/brakeman.ignore generated via:
`brakeman -I  -c config/brakeman.yml`

### CODE REVIEW

**Comment by bf4** [comment]
cc @dwradcliffe

**Comment by dwradcliffe** [comment]
So to update the ignore file we just run that command again?

**Comment by bf4** [comment]
This one

'''
config/brakeman.ignore generated via:
brakeman -I
'''

Have you reviewed the ignore file for if you're ok w ignoring those things?

File: config/brakeman.yml [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
+# :skip_files:
+# - config/database.yml
+# :safe_methods:
+# - :banana
+# :url_safe_methods:
+# - :banana_url
+# :previous_results_json: "./reports/brakeman.json"
+# :output_files:
+# - reports/brakeman.json
+# - reports/brakeman.html
```
Comment (bf4) [inline config/brakeman.yml]: Commented out for reference since config docs can be hard to parse at times.  And some of these settings are actually added by the script

**Comment by dwradcliffe** [comment]
Haven't forgotten about this. I want to get a couple big projects shipped first then we can do this.

**Comment by sonalkr132** [comment]
This PR was too outdated to be worth updating. See #1869
